### PR TITLE
Fix SDN status hook

### PR DESCRIPTION
**Pull request by pravisankar**

### CODE REVIEW

**Comment by pravisankar** [comment]
@danwinship PTAL

File: plugins/osdn/multitenant/plugins.go [hunk 1]
```diff
@@ -69,19 +68,5 @@ func (plugin *MultitenantPlugin) TearDownPod(namespace string, name string, id k
...
-	if err != nil {
-		return nil, err
-	}
-	var podNetworkStatus knetwork.PodNetworkStatus
-	err = json.Unmarshal([]byte(out), &podNetworkStatus)
-	if err != nil {
-		return nil, err
-	}
-	return &podNetworkStatus, nil
+	return &knetwork.PodNetworkStatus{}, nil
```
Comment (danwinship) [inline plugins/osdn/multitenant/plugins.go]: That needs to be "return nil, nil" doesn't it? (look at how it gets used in k8s/pkg/kubelet/dockertools/manager.go)

**Comment by danwinship** [comment]
the rest is good. ironically I just noticed this same problem like 10 minutes ago

**Comment by dcbw** [comment]
LGTM.  Good catch!  Why bother execing when we know what it's going to return :)

**Pull request merged**